### PR TITLE
App notifications

### PR DESCRIPTION
**Pull request by diegomura**

### CODE REVIEW

**Comment by diegomura** [comment]
https://github.com/orbiting/app/pull/95/commits/f4be94aea9e924268fe2b38f3cbc985e2ad35bce thanks!

**Comment by patte** [comment]
@diegomura 
Thanks for the annotations. Foreground and background notifications work fine on iOS as far as my monkey testing goes.
1. let's have a call today to set up your network config. I'll update the readme accordingly.
2. Oh yeah, that looks interesting. I hope we find a better way today.
Cheers

File: src/services/pushNotifications.ios.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import DeviceInfo from 'react-native-device-info'
+import NotificationsIOS from 'react-native-notifications'
+import navigator from './navigation'
+import { setUrl, upsertDevice, rollDeviceToken } from '../apollo'
+
+const pustNotificationsWrapper = WrappedComponent => (
+  class extends Component {
+    componentDidMount () {
+      NotificationsIOS.addEventListener('remoteNotificationsRegistered', this.onPushRegistered)
+      NotificationsIOS.addEventListener('notificationReceivedForeground', this.onNotificationOpened)
```
Comment (patte) [inline src/services/pushNotifications.ios.js]: maybe you could leave a comment about this strange and beautiful solution

File: ios/orbitingapp/AppDelegate.m [hunk 2]
```diff
@@ -120,4 +98,40 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserAct
...
+{
+  [RNNotifications didReceiveLocalNotification:notification];
+}
+
+- (void)userNotificationCenter:(UNUserNotificationCenter *)center willPresentNotification:(UNNotification *)notification withCompletionHandler:(void (^)(UNNotificationPresentationOptions))completionHandler
+{
+  UILocalNotification *localNotification = [[UILocalNotification alloc] init];
+  localNotification.alertTitle = notification.request.content.title;
+  localNotification.alertBody = notification.request.content.body;
+  localNotification.userInfo = notification.request.content.userInfo;
```
Comment (patte) [inline ios/orbitingapp/AppDelegate.m]: maybe you add comment linking to here: https://github.com/wix/react-native-notifications/blob/master/RNNotifications/RNNotifications.m#L353

**Pull request merged**